src/streamr/streamrWsClient.ts

```diff
@@ -62,15 +84,15 @@ export class StreamrWsClient {
       this.subConnection = new WebSocket(
         `${c.readHost}:${c.readPort}/streams/${encodeURIComponent(c.streamId)}/subscribe`
       );
-      this.subStatus = this.getStatusPromise(this.subConnection);
+      this.subStatus = this.getStatusPromise(this.subConnection).then();
     }
     if (c.direction == 'pub' || c.direction == 'all') {
       // eslint-disable-next-line prettier/prettier
       const pubUri = `${c.writeHost}:${c.writePort}/streams/${encodeURIComponent(c.streamId)}/publish?apiKey=${
         c.apiKey
       }`;
       this.pubConnection = new WebSocket(pubUri);
-      this.pubStatus = this.getStatusPromise(this.pubConnection);
+      this.pubStatus = this.getStatusPromise(this.pubConnection).then();
```

Comment:
I believe that the whole idea of creating the async factory method was to move this async line into `create` and `await` there.

Reply:
oops, it shouldn't be here. I'll remove it. :D